Tests/LibMarkdown/CMakeLists.txt

```diff
@@ -8,3 +8,7 @@ set(TEST_SOURCES
 foreach(source IN LISTS TEST_SOURCES)
     serenity_test("${source}" LibMarkdown LIBS LibMarkdown)
 endforeach()
+
+if (BUILD_LAGOM)
+set_tests_properties(TestCommonmark PROPERTIES DISABLED YES)
+endif()
```

Comment:
This is a bit funky, but as far as I can tell is the only way to do this. You can't call set_tests_properties() from another file. https://gitlab.kitware.com/cmake/cmake/-/issues/22813

`BUILD_LAGOM` is maybe not the best way of detecting that we should disable the tests, but it works. :man_shrugging:

Reply:
if(NOT SERENITYOS) would be better, but not sure what that does for building serenity on serenity